satpy/etc/readers/aws_l1b_nc.yaml

```diff
@@ -0,0 +1,415 @@
+reader:
+  name: aws_l1b_nc
+  short_name: AWS L1B RAD NetCDF4
+  long_name: AWS L1B Radiance (NetCDF4)
+  description: Reader for the ESA AWS (Arctic Weather Satellite) Sounder level-1b files in netCDF4.
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+  sensors: [aws,]
+  status: Beta
+  default_channels: []
+
+  data_identification_keys:
+    name:
+      required: true
+    frequency_double_sideband:
+      type: !!python/name:satpy.readers.pmw_channels_definitions.FrequencyDoubleSideBand
+    frequency_range:
+      type: !!python/name:satpy.readers.pmw_channels_definitions.FrequencyRange
+    resolution:
+    polarization:
+      enum:
+        - QH
+        - QV
+    horn:
```

Comment:
What is "horn"? Just curious.

Reply:
From what I understand, the instrument is using different "cameras", or horns, for different channels. Which in turn leads to slightly different longitudes/latitudes for each horn.

Reply:
@djhoese Yes, in MW remote sensing the different radiometer receivers are often called Feedhorns. The Arctic Weather Satellite radiometer has 4 feedhorns, corresponding to the four channel groups, temperature sounding @50-55 GHz, window channel @ 89 GHz, Humidity sounding channels @ 165-183 GHz and the 325 GHz band for humidity and clouds.
We use the same naming for the ICI reader